charts/templates/server-repository/configmap.yaml

```diff
@@ -8,19 +8,16 @@ metadata:
   {{- if .Values.serverRepository.annotations }}
   annotations:
   {{ toYaml .Values.serverRepository.annotations | indent 4 }}
-  {{- end }}    
+  {{- end }}
 data:
-  REPOSITORY_PATH: {{ .Values.serverRepository.config.REPOSITORY_PATH }}
   SERVER_URL: {{ .Values.serverRepository.config.SERVER_URL }}
   REPOSITORY_SERVER_URL: {{ .Values.serverRepository.config.REPOSITORY_SERVER_URL }}
   SERVER_PORT: {{ .Values.serverRepository.service.server.port | quote }}
   SERVER_EXTERNAL_PORT: {{ .Values.serverRepository.service.server.port | quote }}
   REPOSITORY_SERVER_PORT: {{ .Values.serverRepository.service.repository.port | quote }}
   REPOSITORY_SERVER_EXTERNAL_PORT: {{ .Values.serverRepository.service.repository.port | quote }}
```

Comment:
These `*_PORT` config should also be removed, they are not used by any application.